SU-SWS/stanford_profile
# D8CORE-2738 Add filters for event lists

# READY FOR REVIEW

# Summary
- Added taxonomy name contextual argument
- Added card view display using display mode
- https://github.com/SU-SWS/stanford_events/pull/29

# Need Review By (Date)
- 10/30

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. `drush cim`
1. create some events with audience and event type terms associated to them
1. create a basic page with a "Lists" paragraph using the event view
1. Populate the "Arguments" with the taxonomy term names.
   - Note that if you populate it with a string that doesn't correlate to a taxonomy term name, the arguments will be empty and therefore it won't filter the content at all. It will be as if you didn't put any arguments in at all.
1. validate the view is filtered for your content.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
